app/_components/commissionHistory/CommissionHistory.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+interface CommissionItem {
+  id: number;
+  DeveloperName: string;
+  Price: string;
+  code: string;
+  rate:number;
+}
+
+interface CommissionHistoryProps {
+  maxRate?: number;
+}
+
+const CommissionHistory: React.FC<CommissionHistoryProps> = ({ maxRate = 10}) => {
+  const data: CommissionItem[] = [
+    { id: 1, DeveloperName: "Hassan Allam Developer", Price: "30,000", code: "TCR #342131", rate: 9 },
+  ];
+
+  return (
+    <div className="flex justify-between py-6 ">
+      {data.map((item: CommissionItem) => (
+        <>
+        <div key={item.id} className="flex gap-4">
+          <div className="text-[#A1A5B7] leading-6 font-normal text-sm">{`${item.rate}/${maxRate}`}</div>
+          <div className="flex flex-col">
```

Comment:
use this line like this we don't need to concatenate here {item.rate} / {maxRate}